.goreleaser.yml

```diff
@@ -50,3 +50,12 @@ release:
     ```
     docker pull ghcr.io/conduitio/conduit:{{ .Tag }}
     ```
+nfpms:
+  - license: Apache-2.0
+    maintainer: Conduit Developers <conduit-dev@meroxa.io>
+    homepage: https://github.com/ConduitIO/conduit
+    description: "Conduit streams data between data stores. Kafka Connect replacement. No JVM required."
+    section: utils
```

Comment:
for the `section`, I was on the fence between "data-base" and "utils", check https://www.debian.org/doc/debian-policy/ch-archive.html#sections , lemme know what you think

Reply:
`utils` sounds good to me as well. `database` might make someone think it's a database or a database tool (which it kinda is, but not only for databases). I saw `devel` too, but it looks like that's for tools used when developing?

Reply:
yes that's how I imagine `devel` too, or more of a section for developing debian itself